src/site/content/en/blog/value-of-speed/index.md

```diff
@@ -0,0 +1,200 @@
+mobile conversion rate_. This metric is useful because it quantifies the
+effects of site improvements while excluding external factors like marketing
+campaigns, which can obscure your findings. Let's get started!
+
+## Relative Mobile Conversion Rate (Rel mCvR)
+
+A site's conversion rate can be influenced by site characteristics—like speed
+and usability—and by external factors—like marketing campaigns, seasonal events,
+and the mix of marketing channels.
+
+{% Aside %}
+A _conversion_ happens when a site visitor becomes a customer. What counts
+as a conversion will vary depending on the nature of the site. Buying a
+product, subscribing to a service, registering as a user, or even just
+viewing a particular page could be considered a conversion. Check with your
+marketing team if you're not sure how conversions are counted in your company.
+{% endAside %}
+
+Since you're interested in how site speed affects conversions, the mobile site
+is most relevant—that's where you're most likely to see the benefits of
+speed improvements. Rather than looking at the mobile conversion rate alone,
+though, you'll be analyzing the _relative_ mobile conversion rate (Rel mCvr),
+which is calculated by dividing the mobile conversion rate by the desktop
+conversion rate. This approach reduces the noise from external factors, which
+tend to affect both desktop and mobile, and makes it easier to see whether any
+increases in the mobile site's effectiveness were actually caused by the speed
+improvements.
+
+![image](relative-versus-absolute.jpg)
+
+{% Aside 'caution' %}
+Rel mCvR is influenced not only by speed, but also by other site
+characteristics like usability. If other big changes were made to the site
+during the period you want to analyze, you won’t be able to measure the effects
+of speed improvements on their own, but you can show the benefits of all the
+improvements as a group.
+{% endAside %}
+
+## Doing the analysis
+
+If you have access to your site's page analytics data, you can follow along
+using that. In this walkthrough you'll be using Google Analytics to get site
+data. If you don't have a Google Analytics account, you can learn how to set one
+up at [Get started with Analytics](https://support.google.com/analytics/answer/1008015?hl=en).
+Or, if you just want to see how Rel mCvR is calculated, you can get the sample
+data set shown below from
+[this Google Sheet](https://docs.google.com/spreadsheets/d/1Mxmy1luPBOvJneSM9g2NZt6yHKx8xkpBXygOpm4LgNM/edit?usp=sharing)
+and skip steps 1–3.
+
+1. First you'll get the speed and conversion data for the mobile site.
+
+    * Open
+    [this custom report](https://analytics.google.com/analytics/web/template?uid=NPvTU2zoTf2JFsHi6-auyQ)
+    in Google Analytics, select your site from the **Select a view**
+    drop-down list, and click **Create**.
+
+    * Once the report has been generated, click **Select a metric >
+    Metric Group >** **Ecommerce Conversion Rate**.
+    ![image](conversion-rate.jpg)
```

Comment:
For images like this that don't have a clear border we usually switch to using an img tag with the `w-screenshot` class.

```
<img class="w-screenshot" src="./conversion-rate.jpg">
```

Reply:
I also noticed this image has some artifacting. Do you happen to have a higher quality version of this image? Before it gets published our own image optimization scripts will run against it.

Reply:
Done (img tag).

I've asked Lina if she has higher quality versions of the images.